kernel/configs/fragments/x86_64/gpu/version.conf

```diff
@@ -0,0 +1 @@
+CONFIG_LOCALVERSION="-gpu"
```

Comment:
Any reason we need the extra gpu flag? Are we going to ship different kernel? Can we build them as modules and then ship the same kernel package?

Reply:
the modules idea sounds good, but how exactly they will be passed to the agent? in the rootfs/initrd image? through 9p?

Reply:
I don't think this is a widely used scenario, so it can be built separately on demand.

Reply:
since currently we don't have an easy way to share modules with the VM, I think we should at least provide an easy way to allow users build a guest kernel that support specific HW (like GPUs), this doesn't mean that we are going to ship N kernels to cover all scenarios.

Reply:
The guest kernel version of the Nvidia driver compilation and loading environment must correspond, so users can deploy the Nvidia driver module in the image and load the Nvidia driver module in the kata container.
This means that the Nvidia driver module does not have to be included in the guest kernel of kata.
So I think the easy way is to allow VFIO by default and load modules in the guest kernel.

Reply:
> So I think the easy way is to allow VFIO by default and load modules in the guest kernel.

not by default, not all users/workload need VFIO enabled

Reply:
That's why it can be built separately on demand in this PR:)

Reply:
I think `-g` should be enough

Reply:
OK, updated.